src/getHumanAge.test.js

```diff
@@ -7,4 +7,24 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return the Human Age of 15 years old cat and dog', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should return the Human Age of 24 years old cat and dog', () => {
+    expect(getHumanAge(24, 24))
+      .toEqual([2, 2]);
+  });
+
+  test('should return the Human Age of 28 years old cat and dog', () => {
+    expect(getHumanAge(28, 28))
+      .toEqual([3, 2]);
+  });
+
+  test('should return the Human Age of 28 years old cat and 29 dog', () => {
```

Comment:
this test should not work

Reply:
Why? 29 dog years ===  3 Human years and 28 cat years === 3 Human years.